CHANGES.rst

```diff
@@ -85,6 +85,12 @@ photom
   of NIRSpec IFU data with EXTENDED source type; correct units in pixel area
   division to sr from square arcseconds [#7336]
 
+ramp_fitting
+--------
```

Comment:
Still need to fix the underline here, so that it matches the length of the text above.

Reply:
Fixed.